master/buildbot/process/buildstep.py

```diff
@@ -683,21 +696,6 @@ def setupLogfiles(self, cmd, logfiles):
                 # and tell the RemoteCommand to feed it
                 cmd.useLog(newlog, True)
 
-    def interrupt(self, reason):
```

Comment:
This `interrupt` is in `LoggingBuildStep`, which is in fact deprecated.  So I think it makes a lot of sense to move it to the base class.  In short, I like this PR :)